cmd/kured/main.go

```diff
@@ -91,6 +94,12 @@ func main() {
 		PreRun: flagCheck,
 		Run:    root}
 
+	rootCmd.PersistentFlags().BoolVar(&forceReboot, "force-reboot", false,
+		"force a reboot even if the drain is still running (default false)")
+	rootCmd.PersistentFlags().IntVar(&drainGracePeriod, "drain-grace-period", -1,
+		"grace period of time for pods to wait for the node drain in seconds (default -1)")
```

Comment:
it would best to clarify what -1 means: Wait for pod's terminationGracePeriodSeconds (like you rightfully did in drain-timeout)

Reply:
Updated to clarify what happens when the value is negative.